src/components/Rating/Rating.st.css

```diff
@@ -0,0 +1,97 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: color-8;
+    DefaultEmptyColor: color-3;
+    DisabledColor: color-3;
+    IconGap: 3px;
+    IconLargeGap: 11px;
+    IconLargeSize: 30px;
+}
+
+:vars {
+    /* Overrides */
+    IconColor: --overridable;
+    IconEmptyColor: --overridable;
+}
+
+.root {
+    -st-states: mobile;
+
```

Comment:
Maybe we should have the `root` be `inline-block`?
Because currently it's taking the whole width of its parent.
In addition `rtl` isn't working

Reply:
In addition, we probably want to add a `cursor: pointer` on the `.root` when it's not enabled.